templates/search_results.hbs

```diff
@@ -9,18 +9,21 @@
     <section class="search-results-sidebar">
       {{#if help_center_filters}}
         <section class="filters-in-section collapsible-sidebar">
+          <button type="button" class="collapsible-sidebar-toggle" aria-expanded="false"></button>
```

Comment:
Is this empty button needed?

Reply:
yes. without the toggle button we can't expand filters on mobile web. I have uploaded two screenshots to compare in the description of this PR.